src/actions/dataset.ts

```diff
@@ -63,6 +61,8 @@ export function datasetLoad(name: string, dataset: Data): DatasetLoad {
         .then(response => response.json()) // TODO: handle error
         .then(data => fetchCompassQLBuildSchema(data, config)) // TODO: handle error
         .then(schema => {
+          // Clear filters
+          dispatch({ type: FILTER_CLEAR});
```

Comment:
@kanitw I think they should all happen before `DATESET_URL_RECEIVE`, since we should clear the state before a new dataset is loaded. `SHELF_CLEAR` was written before, so I didn't change it. Is there a particular reason it should happen after DATASET_URL_RECEIVE?